src/annotations/components/annotationForm/AnnotationForm.tsx

```diff
@@ -160,12 +159,12 @@ export const AnnotationForm: FC<Props> = (props: Props) => {
   const isTimeInFuture = (timeToCheck): boolean => {
     let now = null
     if (props.getNow) {
-      now = moment(props.getNow())
+      now = new Date(props.getNow())
     } else {
-      now = moment()
+      now = new Date()
     }
 
-    return moment(timeToCheck).isAfter(now)
+    return new Date(timeToCheck).getTime() > now.getTime()
```

Comment:
looks good; but since 'isAfter' is used twice; please put in a method!  (DRY)

Reply:
Got it! Will do 👍

Reply:
Done ✅